src/Analysis/Ast/Impl/Analyzer/Symbols/ClassEvaluator.cs

```diff
@@ -16,6 +16,7 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
+using System.Runtime.CompilerServices;
```

Comment:
Remove?

Reply:
(A few usings are unused, the document could use an autoformat)

Reply:
Ok.